### DISCUSSION TITLE
cn/2024/01/data-visualization-tips/

### DISCUSSION THREAD

**Discussion by giscus** [leovan/leovan.me#40] [Comments]
# cn/2024/01/data-visualization-tips/

文本主要面向不同格式文档（HTML、PDF、Word）的动态生成，秉承规范、统一和实用的理念总结数据可视化过程中的相关问题，不过度涉及数据可

https://leovan.me/cn/2024/01/data-visualization-tips/

**Comment by GavinCruise** [comment]
路过点个赞，很好的文章。
